### PR TITLE
support for auto-privacy filter settings

### PR DESCRIPTION
**Pull request by yamini**
- Change privacy filter settings in default, high-dimensionality, low-record count, and high-accuracy blueprints. 
- Rename dgan.yml to time-series.yml

### CODE REVIEW

File: config_templates/gretel/synthetics/default.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Default configuration for Synthetic model creation.
 
 # The parameter settings below match the default settings 
-# in Gretel's open source synthetic package
+# in Gretel's open source synthetics package
```
Comment (johntmyers) [inline config_templates/gretel/synthetics/default.yml:4]: Nit: suggest changing this to SAL language

Comment (yamini) [reply config_templates/gretel/synthetics/default.yml]: done

**Review by johntmyers — APPROVED** [review]

**Review by sami-t — COMMENTED** [review]
Looks great to me, thanks Yamini!

**Comment by sami-t** [comment]
Actually we also need to rename the config in the DGAN blueprint notebook: https://github.com/gretelai/gretel-blueprints/pull/134

**Pull request merged**